tests/packages/install-subdir-exclude/meson.build

```diff
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: 2023 The meson-python developers
+#
+# SPDX-License-Identifier: MIT
+
+project('install-subdir-exclude', version: '1.0.0')
+
+py = import('python').find_installation()
+purelib = py.get_install_dir(pure: true)
+
```

Comment:
Why doesn't this test include the use of `exclude_directories` and `exclude_files`? That should be the main point, right? More interesting than the niche bug reported in meson-11539.